css/mirador.less/panels/bottom-panel.less

```diff
@@ -38,6 +38,12 @@
         padding: 0px 10px 0px 10px;
         /*transition: all 0.03s ease-out;*/
         display: inline-block;
+        margin-bottom: 11px !important;
+
+        &.highlight {
+          padding-left: 8px !important;
+          padding-right: 8px !important;
```

Comment:
These rules containing `!important` came in with the commit 9f23360 by @rsinghal. Is it really necessary, to keep the `!important`? I just tested it without and nothing changed.